CMakeLists.txt

```diff
@@ -29,11 +29,6 @@ set(OPENSSL_VERSION_NUMBER  0x11100000L)
 # Generates compile_commands.json file for clangd to parse.
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
-if ($ENV{XP_BUILD})
```

Comment:
Don't we want to remove `XP_BUILD` support as well?

Reply:
hmm good question. Now XP build is using a separated repo so maybe we could remove `XP_BUILD` from all the build scripts? I think better to leave `XP_BUILD` unchanged for now to be safe, we could have a separated PR for that once confirmed.